paasta_tools/cli/cmds/spark_run.py

```diff
@@ -894,6 +992,18 @@ def paasta_spark_run(args):
     user_spark_opts = _parse_user_spark_args(
         args.spark_args, pod_template_path, args.enable_compact_bin_packing
     )
+
+    # Get driver-memory and cores
+    sub_cmds = args.cmd.split(" ")  # spark.driver.memory=10g
```

Comment:
this also seems a little brittle, but that's a problem for future-us I guess (something like providing native soaconfigs support for this would be nice)

Reply:
I had added the detailed comment at line #993-994. I will move it upwards. Hopefully, that is more clearer

Reply:
> just double-checking: there's no values for any of these config options that could potentially have a space in them?
This is correct. In fact I could remove this too. There is no real use-case at Yelp being used in this.
I added it to take care of future possible edge cases.